Sources/XCTest/Public/XCTestMain.swift

```diff
@@ -59,19 +59,13 @@
 ///
 /// - Parameter testCases: An array of test cases run, each produced by a call to the `testCase` function
 /// - seealso: `testCase`
-public func XCTMain(_ testCases: [XCTestCaseEntry]) -> Never {
-    XCTMain(testCases, arguments: CommandLine.arguments)
-}
-
-public func XCTMain(_ testCases: [XCTestCaseEntry], arguments: [String]) -> Never {
-    XCTMain(testCases, arguments: arguments, observers: [PrintObserver()])
-}
-
+@_disfavoredOverload
 public func XCTMain(
```

Comment:
DocC documentation for the new overload? Its shape looks good to me, though

Reply:
The new overload has DocC. This is now the old one.

Reply:
Cleaned up docs for the new overload.